optimade/server/routers/versions.py

```diff
@@ -9,6 +10,18 @@
 class CsvResponse(Response):
     media_type = "text/csv"
 
+    def init_headers(self, *args, **kwargs):
+        """ Patch for base class `init_headers` that adds the
+        `header=present` content-type parameter without affecting
+        the OpenAPI schema.
+
+        """
+        cached_charset = copy.deepcopy(self.charset)
+        self.charset = "{self.charset}; header=present"
```

Comment:
Will `self.charset` ever be empty? If yes, then this should be changed, I think?

Reply:
Nope, it's hardcoded as "utf-8" by FastAPI for the base class, then modified if necessary. I would rather not try to pre-empt future changes to FastAPI, we want it to be very clear if this hack fails for future versions (or even request a cleaner way of doing this). Let's wait to see what the others say.